tests/DCPS/MultiTopic/MultiTopicTest.idl

```diff
@@ -1,49 +1,75 @@
 // example adapted from the DCPS Specification v1.2, Annex A, Section 3
 
-#pragma DCPS_DATA_TYPE "LocationInfo"
-#pragma DCPS_DATA_KEY "LocationInfo flight_id1"
-#pragma DCPS_DATA_KEY "LocationInfo flight_id2"
+@topic
 struct LocationInfo {
-  unsigned long flight_id1;
-  unsigned long flight_id2;
+  @key unsigned long flight_id;
+  @key unsigned long departure_date;
   long x;
   long y;
   long z;
 };
 
-#pragma DCPS_DATA_TYPE "PlanInfo"
-#pragma DCPS_DATA_KEY "PlanInfo flight_id1"
-#pragma DCPS_DATA_KEY "PlanInfo flight_id2"
+enum FlightType {
+  visual,
+  instrument,
+  mixed
+};
+
+typedef string<3> Airport; // 3 Letter IATA Code
+typedef sequence<Airport> Airports;
```

Comment:
https://opendds.org/faq.html#idl-builtin
This is why the MSVC builds are failing on Azure CI.

Reply:
Alright, but how does this work for a bounded string?

Reply:
I'm not sure, we might have to do so more to get it working.  But I'd start with `#include`ing TAO's IDL file since we know that solves some form of this problem.

Reply:
Huh, I don't think it should've worked if it's a bounded string, but I guess I can't really complain if it worked.